cmd/test/main.go

```diff
@@ -15,7 +15,8 @@ func main() {
 
 	options := options.Generic{"AddressIPv4": net}
 	controller := libnetwork.New()
-	netw, err := controller.NewNetwork("simplebridge", "dummy", options)
+	driver, _ := controller.NewNetworkDriver("simplebridge", options)
+	netw, err := controller.NewNetwork(driver, "dummy", "")
```

Comment:
Same here. when we have the plugin<->network binding feature available, we should ideally move the driver/plugin selection into the libnetwork's NewNetwork method and abstract it out of the application.
Hence it is better to add a comment to reflect that.